lms/static/scripts/frontend_apps/components/DeepLinkingFilePickerFormFields.tsx

```diff
@@ -0,0 +1,27 @@
+export type DeepLInkingFilePickerFormFieldsProps = {
+  /**
+   * Form field values provided by the backend that should be rendered as
+   * hidden input fields.
+   * These are used while deep linking and it should be the only data sent to the LMS.
+   */
+  deepLinkingFields: Record<string, string>;
+};
+
+/**
+ * Render the hidden form fields in the file picker form containing information
+ * about the selected assignment.
+ *
+ * Used when  deep linking an assignment. The form will be submitted to the LMS
+ * with only the data provided by the backend in `deepLinkingFields`.
+ */
+export default function DeepLinkingFilePickerFormFields({
+  deepLinkingFields,
+}: DeepLInkingFilePickerFormFieldsProps) {
```

Comment:
```suggestion
}: DeepLinkingFilePickerFormFieldsProps) {
```

The "i" should be lower-case.